packages/protocol/contracts/governance/ReleaseGoldInstance.sol

```diff
@@ -198,6 +199,7 @@ contract ReleaseGoldInstance is UsingRegistry, ReentrancyGuard, IReleaseGoldInst
 
     setRegistry(registryAddress);
     beneficiary = _beneficiary;
+    emit BeneficiarySet(beneficiary);
```

Comment:
Please define a `_setBeneficiary` that doesn't have the modifier, and then use that here, and then `setBeneficiary` just wraps `_setBeneficiary` with the modifier